api/api/controllers/search_controller.py

```diff
@@ -560,19 +554,19 @@ def get_sources(index):
     except ValueError:
         cache_fetch_failed = True
         sources = None
-        log.warning("Source cache fetch failed due to corruption")
+        logger.warning("Source cache fetch failed due to corruption")
     except ConnectionError:
         cache_fetch_failed = True
         sources = None
-        log.warning("Redis connect failed, cannot get cached sources.")
+        logger.warning("Redis connect failed, cannot get cached sources.")
```

Comment:
I wonder if it would be nice to change these to structured logs too, maybe in a future issue. Something like `logger.warning("Redis connect failed", when="get-cached-sources")`, and so forth. Ideally these logs never matter though, and would never be meaningful to query, so probably not worth it!

Reply:
Yes, we should definitely update existing logs to be more "structured" but that, like you said, will have to be a separate issue and PR.